module-1/pipelines-project/your-code/pipeline_miriam.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd 
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+def acquire():
+    data = pd.read_csv('data/blackfriday.csv')
+    return data
+
+def wrangle(data, year):
+    filtered = data[data['year']==year]
+    return filtered
+
+def renamed(data):
+    column_name = filtered.rename(columns={'Purchase':'Expenditure', 'City_Category': 'City'})
+    return column_name
+
+def drop_columns (data):
+    new_data = column_name.drop(column_name.columns[[6]], axis=1)
+    return new_data
+
+def column_order (data):
```

Comment:
There are other optimum ways of changing column orders. This is affordable when you have a small number of columns otherwise you need to try other alternatives. 
You can use some of the following 
```Python
df = df.ix[:, cols] or cols = cols[-1:] + cols[:-1]
then df = df[cols]
```
We can take a look on it when you find a slot in your busy agenda!!